FirebaseRemoteConfig/Tests/Unit/RCNRemoteConfigTest.m

```diff
@@ -112,14 +114,14 @@ - (void)setUp {
 
   // Always remove the database at the start of testing.
   _DBPath = [RCNTestUtilities remoteConfigPathForTestDatabase];
-  id classMock = OCMClassMock([RCNConfigDBManager class]);
-  OCMStub([classMock remoteConfigPathForDatabase]).andReturn(_DBPath);
+  id _DBManagerMock = OCMClassMock([RCNConfigDBManager class]);
```

Comment:
Did you mean using ivar instead of local var here?

Reply:
Ack. Done.